tests/draft3/ref.json

```diff
@@ -125,7 +125,7 @@
                 "b": {"$ref": "#/definitions/a"},
                 "c": {"$ref": "#/definitions/b"}
             },
-            "$ref": "#/definitions/c"
+            "allOf": [{"$ref": "#/definitions/c"}]
```

Comment:
IIRC allOf isn't in draft 3.

Reply:
... and that's another good reason to close this.  I forgot that draft-03 had `extends` or something.